README.oscal-models-upgrade.md

```diff
@@ -0,0 +1,40 @@
+## Developers Guide to trestle upgrade commensurate with OSCAL models upgrade
```

Comment:
I think the content of this is really helpful, but I am wondering if it would make more sense to add to the contributing guide or at least reference it in the contributing guide with the other developer guide information.

Reply:
This developer info is kind of specific to trestle. I fine with moving, but where exactly to?

Reply:
Agreed. I meant the compliance-trestle `CONTRIBUTING.md`. Maybe link it somewhere around here - https://github.com/oscal-compass/compliance-trestle/blob/031850f91a83f6fdd569025982a923cd10123938/CONTRIBUTING.md?plain=1#L131

Reply:
Sure, seems reasonable. Moved.